### PR TITLE
UI revamp changes

### PR DESCRIPTION
**Pull request by satyambnsal**
Changes summary

#147 
#148 
#149 
#150 
#151 
#114 
#38

### CODE REVIEW

**Comment by varex83** [comment]
image

I have some problems with paddings/margin here, can you fix it to be centered?

**Comment by varex83** [comment]
image

also, remove this shadow under the env section

**Comment by varex83** [comment]
image

nitpick, make it more table-like, and use tx hash instead of txid

**Comment by varex83** [comment]
image

can we have the same width of dropdown as selector?

File: plugin/src/components/Settings/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import './settings.css'
+
+const DEFAULT_DELAY = 5_000
+
+export const Settings = () => {
+  const remixClient = useAtomValue(remixClientAtom)
+  const [solidityVersion, setSolidityVersion] = useAtom(solidityVersionAtom)
+  const [versions, setVersions] = useAtom(versionsAtom)
+
+  const [, refetchVersions] = useAsyncFn(async () => {
```
Comment (varex83) [inline plugin/src/components/Settings/index.tsx]: with this approach, we are loading versions only when rendering "Settings" tab, so each time we need to go here and wait until plugin fetch it, but it should done as soon as plugin loads

File: plugin/src/features/CompilerVersion/index.tsx [hunk 2]
```diff
@@ -1,26 +1,16 @@
-import React, { useEffect } from 'react'
```
Comment (varex83) [inline plugin/src/features/CompilerVersion/index.tsx:1]: Maybe we should rename `CompilerVersion` to something like `Header` or smth

**Review by varex83 — CHANGES_REQUESTED** [review]

File: plugin/src/components/Settings/index.tsx (hunk 1 above)
Comment (satyambnsal) [reply plugin/src/components/Settings/index.tsx]: Ah good catch!!. Thank you. I moved it to CompilerVersion component so it loads on initial render.

**Comment by satyambnsal** [comment]
@varex83 Thank you for the thorough review. I've addressed all the points raised, with one exception: changing the Transaction history table structure to a more table-like format. This modification would require good CSS changes, so I propose we handle this in a separate PR.
Here is how it looks after changing TxId to TxHash

Screenshot 2024-07-26 at 3 50 46 PM

**Comment by satyambnsal** [comment]
Review changes done.

**Review by varex83 — CHANGES_REQUESTED** [review]
image

can you quickly add some margings there, and we are good to go?

**Review by varex83 — APPROVED** [review]
LGTM

**Pull request merged**